antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4

```diff
@@ -20,7 +20,7 @@
 grammar SqlBase;
 
 singleStatement
-    : statement EOF
+    : statement (';')? EOF
```

Comment:
need to update docs?

Reply:
Actually, it is the doc shows we can support the sql statement end with semicolon...

See
https://github.com/apache/iotdb/blob/master/docs/UserGuide/Client/Programming%20-%20JDBC.md#examples